draft.md

```diff
@@ -84,14 +84,44 @@ The idempotency key that is supplied as part of every `POST` request MUST be uni
 
 How to make the key unique is up to the client and it's agreed protocol with the server. It is `RECOMMENDED` that `UUID` or a similar random identifier be used as the idempotency key.
 
+## Idempotency Key Validity and Expiry
 
-## Resource Behavior
+The server MAY enforce a time based idempotency keys, thus, be able to purge or delete a key upon its expiry. Resource SHOULD publish expiration policy related documentation.
```

Comment:
Can we stick to one term? Server or resource. I prefer the latter since it narrows the scope which is good for ID (Internet Draft)

Reply:
we can stick with `Resource Server` if you are ok. I also dont have any reservations about `Resource` so we can do that as well. Let me know

Reply:
In notational conventions section, we refer to "resource" as described in https://tools.ietf.org/html/rfc7231#section-2. Let's stick to that term since we are in IETF and HTTP land.